### PR TITLE
Update design of runs pages

### PR DESCRIPTION
**Pull request by AlanGreene**
# Changes

Update the table layout used for the PipelineRuns and TaskRuns pages
to include more useful information such as trigger info, status details,
and combine fields where this makes sense (e.g. created time + duration).

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [ ] [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
- [x] [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [x] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [x] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including
  functionality, content, code)

_See [the contribution guide](https://github.com/tektoncd/dashboard/blob/main/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: packages/components/src/components/PipelineRuns/PipelineRuns.js [hunk 1]
```diff
@@ -91,27 +110,16 @@ const PipelineRuns = ({
...
-    }),
     pipeline: intl.formatMessage({
       id: 'dashboard.tableHeader.pipeline',
       defaultMessage: 'Pipeline'
     }),
-    namespace: 'Namespace',
-    createdTime: intl.formatMessage({
-      id: 'dashboard.tableHeader.createdTime',
-      defaultMessage: 'Created'
+    run: 'Run',
```
Comment (briangleeson) [inline packages/components/src/components/PipelineRuns/PipelineRuns.js:117]: Just to confirm, should this be extracted or treating as a special word that is never translated?

File: packages/components/src/components/TaskRuns/TaskRuns.js [hunk 2]
```diff
@@ -71,20 +102,17 @@ const TaskRuns = ({
 }) => {
   let hasRunActions = false;
   const headers = [
+    {
+      key: 'run',
+      header: 'Run'
```
Comment (briangleeson) [inline packages/components/src/components/TaskRuns/TaskRuns.js:107]: Same question again about extracting the string

File: packages/components/src/components/Table/Table.js [hunk 3]
```diff
@@ -117,19 +117,19 @@ const Table = props => {
     emptyTextAllNamespaces,
     emptyTextSelectedNamespace,
     filters,
+    hasDetailRows,
```
Comment (briangleeson) [inline packages/components/src/components/Table/Table.js:120]: You had asked about the naming convetion here. Only other thought was `hasDetails`

Alternatively scrub the extra `hasDetailRows` prop and just set the `size` prop to `xl` for the Table component in PipelineRuns and TaskRuns? I dont recall from the walkthrough if you had a case in mind that meant the extra prop was needed

**Review by briangleeson — COMMENTED** [review]
Looks good, and cheers for the walkthrough

Couple of minor questions, but otherwise good to go

File: packages/components/src/components/PipelineRuns/PipelineRuns.js (hunk 1 above)
Comment (AlanGreene) [reply packages/components/src/components/PipelineRuns/PipelineRuns.js:117]: special word, similar to 'PipelineRun' 👍

File: packages/components/src/components/Table/Table.js (hunk 3 above)
Comment (AlanGreene) [reply packages/components/src/components/Table/Table.js:120]: I prefer `hasDetails`, I'll update it. Thanks.

Yes it's needed to control positioning of the batch action checkboxes + overflow menu icons as they're not vertically centered in this case as they would be by just setting `size="xl"`, as well as some other tweaks to spacing of the content (see the `tkn--table-with-detail-rows` class which I'll also rename)

Comment (AlanGreene) [reply packages/components/src/components/Table/Table.js]: Updated 👍

**Review by briangleeson — APPROVED** [review]
/lgtm

**Pull request merged**